CMakeLists.txt

```
@@ -38,8 +38,12 @@ if (QUICKCPPLIB_REQUIRE_CXX17 OR QUICKCPPLIB_REQUIRE_CXX20)
 endif()
 # Make an interface only library so dependent CMakeLists can bring in this header-only library
 include(QuickCppLibMakeHeaderOnlyLibrary)
-# Make a docs target
-include(QuickCppLibMakeDoxygen)
+
+set(quickcpplib_BUILD_DOCS ON)
```

```suggestion
option(quickcpplib_BUILD_DOCS "Enable the doxygen target" ON)
```
`set` shadows cache variables, i.e. this option couldn't be set via the cmake commandline/gui